ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -1718,7 +1733,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ $SDKROOT == *\"iPhoneSimulator\"* ]]; then\necho [Bundle React Native code and images] Skipping sentry\n../node_modules/react-native/scripts/react-native-xcode.sh\nelse\nexport SENTRY_PROPERTIES=sentry.properties\nexport NODE_BINARY=node\n../node_modules/@sentry/cli/bin/sentry-cli react-native xcode ../node_modules/react-native/scripts/react-native-xcode.sh\nfi\n";
+			shellScript = "if [[ $CONFIGURATION == \"Release\" && ! `git describe --tags` =~ '-g' ]]; then\n  export SENTRY_PROPERTIES=sentry.properties\n  export NODE_BINARY=node\n  ../node_modules/@sentry/cli/bin/sentry-cli react-native xcode \\\n    --force-foreground \\\n    ../node_modules/react-native/scripts/react-native-xcode.sh\nelse\n  echo [Bundle React Native code and images] Skipping sentry\n  ../node_modules/react-native/scripts/react-native-xcode.sh\nfi\n";
```

Comment:
From in-person discussion: Sentry post-install scripts munge these (and for one of the two that's even quite legitimate because Sentry created the step in the first place), so a cost of moving them out is that it may make it harder to tell what future Sentry upgrades are trying to do.

OTOH it sounds like that was already a pretty from-scratch merge experience with the current strategy -- so we may be paying much the same cost already just by having our own changes here.